scipy/signal/tests/test_ltisys.py

```diff
@@ -233,7 +345,6 @@ def test_errors(self):
         assert_raises(ValueError, place_poles, A, B,
                       (-2+1j,-2-1j,-2+3j,-2-3j), method="KNV0")
 
-
```

Comment:
this should probably still be here for PEP8 (two lines before class or function def at root indentation)